test-site/src/components/HTMLExampleCard.tsx

```diff
@@ -16,13 +16,15 @@ const htmlFieldName = 'c_richTextV2Data';
 interface CustomRawDataType {
   name: string,
   description: string,
-  [htmlFieldName]: any
+  [htmlFieldName]: { html: string }
 }
 
 function renderHTMLContent(HTMLContent: string | undefined) {
   if ( HTMLContent )
   {
-    return <div className="reset-style" dangerouslySetInnerHTML={{ __html: HTMLContent }} />;
+    const createDangerousHTMLObject = (htmlContent: string) => ({ __html: htmlContent });
+    const dangerouslySetHTML = createDangerousHTMLObject(HTMLContent);
+    return <div className="reset-style" dangerouslySetInnerHTML={dangerouslySetHTML} />;
```

Comment:
Fixing the linting error seems to have complicated this a bit. Can we instead create the dangerous html object in the HTML example card using the useMemo hook and pass that object into the renderHTML content function? While we're here, can we also avoid using the parameter name HTMLContent since that looks like a type rather than a variable? I think we should stick to `htmlContent` or `html`